core/services/capreg/types.go

```diff
@@ -0,0 +1,151 @@
+package capreg
+
+import (
+	"bytes"
+	"context"
+	"math/big"
+)
+
+// CapabilityID is the unique identifier of the capability in the CR.
+// It is calculated as keccak256(abi.encode(capabilityType, capabilityVersion)).
+type CapabilityID = string
+
+// DON represents a DON in the Capability Registry.
+type DON struct {
```

Comment:
There is a DON struct in common: https://github.com/smartcontractkit/chainlink-common/blob/main/pkg/capabilities/capabilities.go#L161

How do we reconcile these two? @cedric-cordenier thoughts?